### ISSUE TITLE
Usermenu support

### ISSUE THREAD

**Issue by ghost** [fairyglade/ly#373]
How about reading user list directly from `/home/` & give us those list in the form of left+right switchable menu (similar to 1st row, desktop environment)

Then you can have the pointer directly on password textbox with a default chosen user or 1st user from the list. Makes life lil easy eliminiating the need to enter username too.

**Comment by Cavernosa** [comment]
I don't think it would be possible to automatically get users (i.e. root is in /root, not /home/root) but you can make ly remember your username and DE so you only have to type the password. `save = true`

**Comment by pc00per** [comment]
> I don't think it would be possible to automatically get users (i.e. root is in /root, not /home/root) but you can make ly remember your username and DE so you only have to type the password. `save = true`

Makes sense. That's the default behaviour for now. But I think it's still possible. `/root/` doesn't mean you don't have permissions for other `/` directories. I've checked `ls /home/` from root login & I can successfully able to see all of my user accounts list.

**Comment by Cavernosa** [comment]
I meant that it could be hard to implement if someone had users all over the place, outside of /home, but now that i think about it yeah it might be possible,  since the only (important) user that is not in /home is root.
Alternatively it could save the user to a list when someone logins with a new user like some other DMs do

**Comment by pc00per** [comment]
> Alternatively it could save the user to a list

You don't even need to. Just `ls` the `/home/` & put those directories as menu list on an ly column.
For last default used user, ly already does that now afaik.
